en/docs/install-and-setup/setup/deployment-best-practices/security-guidelines-for-production-deployment.md

```diff
@@ -255,6 +255,13 @@ been removed from Hotspot JVM.</p>
 <td><p>For password recovery, you can define a regular expression to validate the callback URL. The default configuration allows any callback URL. Note that if you are using the recovery option, it is highly recommended to define the regular expression that validates and only allows access to specific callback URLs.</p>
 </td>
 </tr>
+<tr class="odd">
```

Comment:
There is another section called "Enable HostName Verification" in the same table. I thin we can use it instead of creating a new row. And this already existing row points to [1] which we can make use of here. Then I think no need to add the information in this doc. WDYT?

[1] https://apim.docs.wso2.com/en/latest/install-and-setup/setup/security/enabling-hostname-verification/

Reply:
+1. Will remove this. It's better to only mention in one place anyway.